resources/definitions/fdmprinter.def.json

```diff
@@ -3379,6 +3379,30 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "support_skip_some_zags":
+                {
+                    "label": "Skip Some ZigZags Connections",
+                    "description": "Skip some ZigZags connections to make the support structure easier to break.",
+                    "type": "bool",
```

Comment:
I think you can leave out this whole setting by making a special case for support_zag_skip_count. You could make it such that `if support_zag_skip_count == 0 then don't skip any connections`.
The description of support_zag_skip_count would then also need to state that this works that way.

Please discuss with Ruben what you think is best.